docker-files-for-Gitlab-CI-rust/parity-eth-linux-ci/Dockerfile

```diff
@@ -1,18 +1,27 @@
-FROM rust:stretch
-LABEL maintainer="Parity Technologies <devops-team@parity.io>"
-LABEL description="This is for CI build stage for Parity Ethereum linux binary."
+# triggers image autoupdate with the new Rust release
+FROM rust:slim as notifier
+
+# temporary support of 16.04 with its Clib
+FROM ubuntu:xenial as builder
+
+# metadata
+LABEL maintainer="devops-team@parity.io" \
+	vendor="Parity Technologies" \
+	name="parity/rust-parity-ethereum-build" \
+	description="This is for CI build stage for Parity Ethereum linux binary." \
+	vcs-url="https://gitlab.com/paritytech/scripts/docker-files-for-Gitlab-CI-rust/parity-eth-linux-ci" \
+	url="https://gitlab.com/paritytech"
 
 # install tools and dependencies
 RUN apt-get -y update && \
 	apt-get install -y --no-install-recommends \
 	software-properties-common curl git \
 	make cmake ca-certificates g++ rhash \
-	gcc pkg-config libudev-dev time
-
-# removed:
-# binutils binutils-dev snapcraft gettext file python build-essential zip dpkg-dev rpm libssl-dev openssl ruby-dev
+	gcc pkg-config libudev-dev time \
+	libssl-dev
```

Comment:
why did you add libssl back in?

Reply:
unfortunately, it's required for installations of `sccache` and `audit` in `ubuntu:xenial`, and in `debian:stretch` it's just built-in.